Automattic/themes
# Vetro: use latest duotone features

#### Changes proposed in this Pull Request:

This PR applies duotone via global styles instead of directly in the template. It also restores the duotone effects to its original intentions with each style variation. 

#### To test:

- Activate Vetro
- Switch between style variations (I noticed there is a delay for the duotone to apply, seems like the filter is cached for about a minute)
- Verify the images appear as expected with the correct duotone filters

cc @henriqueiamarino
